Automattic/wp-calypso
# Docs: Add a graphical representation of commands executed with yarn start

## Proposed Changes

* Add a section to the document `Tech behind Calypso` with a graphical representation of the commands executed when running `yarn start`

## Testing Instructions



* Apply this patch
* Navigate to `/devdocs/docs/guide/tech-behind-calypso.md`
* Scroll to bottom and check the new section named `Commands executed when you run yarn start`
  ![CleanShot 2023-11-16 at 16 02 10@2x](https://github.com/Automattic/wp-calypso/assets/3519124/fb43e6e5-7f9b-474a-b36a-1749366ac646)

* Note: the `mermaid` diagram doesn't work in Calypso docs but it renders when being displayed by Github so the link to GH points to `trunk` and it will only work when merging this PR.

## Pre-merge Checklist
